catalog/security/catalog-security-plugin/src/main/java/ddf/catalog/security/plugin/SecurityPlugin.java

```diff
@@ -1,10 +1,10 @@
 /**
```

Comment:
This class has no unit tests... is that OK? Should we take the time to create some? It's pretty small/simple...